docs/get-started/alerting.md

```diff
@@ -74,6 +171,7 @@ If none of the default PMM templates contain a relevant expression for the alert
 You can base multiple alert rules on the same template. For example, you can create a `pmm_node_high_cpu_load` template that can be used as the source for alert rules for production versus staging, warning versus critical, etc.
```

Comment:
Another example would be that you could create a HIGH CPU alert from our template that has 30% threshold and displays as 'warning', and create another alert, using the same template, but modify this one to use a different threshold at 50%, categorized as critical